api/api_test.go

```diff
@@ -18,11 +18,23 @@ func TestSetup(t *testing.T) {
 		ctx := context.Background()
 
 		mockMongoDB := &mock.DataStoreMock{}
-		cacheAPI := api.Setup(ctx, router, mockMongoDB)
+		isPublishing := true
```

Comment:
This shouldn't be here, it makes more sense to be inside "When created in publishing mode".